charts/mattermost-operator/templates/mysql-operator/secret.yaml

```
@@ -0,0 +1,13 @@
+{{- if and .Values.mysqlOperator.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "mysql-operator.name" . }}-orc
+  namespace: {{ .Values.mysqlOperator.namespace }}
```

Just to be inline with all others:
```suggestion
  namespace: {{ template "mysql-operator.namespace" . }}
```